### PR TITLE
Update beneficiary name requirements

### PR DESCRIPTION
**Pull request by hitmanmcc**
This PR updates the requirements for the beneficiary name property in transactions.

### CODE REVIEW

File: _transactions.md [hunk 1]
```diff
@@ -368,31 +368,31 @@ As such, under the regulatory action of The Financial Action Task Force [FATF](h
...
 
-Parameter | Required | Description
---------- | ----------- | -----------
-address | yes/no | The transaction beneficiary address information. See [Address](#address). <br><br> <b>Required</b> for invites and external beneficiaries.
-name | yes/no | The beneficiary full name. <br><br> <b>Required</b> for invites and external beneficiaries.
-relationship | yes | Reflects the beneficiary's relationship to the transaction originator. <br><br> Possible values are `business`, `child`, `co_worker`, `friend`, `myself`, `parent`, `sibling`.
+Parameter    | Required    | Description
+------------ | ----------- | -----------
+address      | yes/no      | The transaction beneficiary address information. See [Address](#address). <br><br> <b>Required</b> for invites and external beneficiaries.
+name         | yes/no      | The beneficiary's full name. <br><br> <b>Required</b> for invites and external beneficiaries. <br><br> For all transactions, except those with `relationship` type `business`, the name must be composed of at least, 2 words with a minimum of 2 characters each, for the first and last word.
```
Comment (pgom) [inline _transactions.md:383]: `the first and last name`?

Comment (hitmanmcc) [reply _transactions.md:383]: Whilst not incorrect, I believe the current version makes for a more cohesive explanation as we refer that there need to be at least 2 words with a minimum of 2 characters. So if you go beyond to words, then the rule applies to the first and the last word in the string.

Comment (pgom) [reply _transactions.md:383]: Fair enough

**Review by pgom — APPROVED** [review]

File: _transactions.md (hunk 1 above)
Comment (waldyrious) [reply _transactions.md:383]: @hitmanmcc the comma after "at least" breaks the sentence in an awkward manner, IMHO. I'd suggest either removing it:

> ...composed of at least 2 words...

or adding another comma before the expression:

> ...composed of, at least, 2 words...

**Pull request merged**